ext/concurrent/ruby_193_compatible.h

```diff
@@ -17,12 +17,12 @@ static inline void rb_error_arity(int argc, int min, int max)
   else {
     err_mess = rb_sprintf("wrong number of arguments (%d for %d..%d)", argc, min, max);
   }
-  rb_raise(rb_eTypeError, err_mess);
+  return rb_ext_new3(rb_eTypeError, err_mess);
```

Comment:
I'm surprised the Travis build succeeded. It looks like the extension is being compiled with very lenient settings, to the extent that calling non-existent functions isn't even considered an error!